### ISSUE TITLE
Fix the IDs being returned by the Loris responses

### ISSUE THREAD

**Issue by alexwlchan** [wellcometrust/platform-api#467]
Right now, hitting https://iiif.wellcomecollection.org/image/V0000001.jpg/info.json returns the following:

```json
{
  "profile": [
    "http://iiif.io/api/image/2/level2.json",
    {
      "supports": [
        "canonicalLinkHeader",
        "profileLinkHeader",
        "mirroring",
        "rotationArbitrary",
        "regionSquare",
        "sizeAboveFull"
      ],
      "qualities": [
        "default",
        "color",
        "gray",
        "bitonal"
      ],
      "formats": [
        "jpg",
        "png",
        "gif",
        "webp"
      ]
    }
  ],
  "protocol": "http://iiif.io/api/image",
  "sizes": [

  ],
  "height": 3432,
  "width": 2814,
  "@context": "http://iiif.io/api/image/2/context.json",
  "@id": "https://app:8888/s3%3AV0000000%2FV0000001.jpg"
}
```

We should hide the `app:8888` part.

**Comment by jtweed** [comment]
Now coming through as `http://iiif-origin.wellcomecollection.org/` which is still wrong, but in a different way.

**Comment by alexwlchan** [comment]
Resolved by #885. We had to flush the CloudFront cache to fix up any `info.json` requests that were cached with the old, wrong IDs – but now all seems to be working.
